### PR TITLE
flesh out project structure

### PR DESCRIPTION
**Pull request by asphaltbuffet**

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+linters-settings:
+  gocyclo:
+    min-complexity: 15
+  goimports:
+    local-prefixes: github.com/asphaltbuffet/passphrase
```
Comment (asphaltbuffet) [inline .golangci.yml]:
```suggestion
    local-prefixes: github.com/milligan22963/passphrase
```

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "[go.mod]": {
+        "editor.formatOnSave": true,
+        "editor.codeActionsOnSave": {
+            "source.organizeImports": true,
+        },
+    },
+    // gopls
+    "go.useLanguageServer": true,
+    "gopls": {
+        "formatting.local": "github.com/asphaltbuffet/passphrase",
```
Comment (asphaltbuffet) [inline .vscode/settings.json]:
```suggestion
        "formatting.local": "github.com/milligan22963/passphrase",
```

File: .golangci.yml (hunk 1 above)
Comment (asphaltbuffet) [inline .golangci.yml]:
```suggestion
    local-prefixes: github.com/milligan22963/passphrase
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at **TBD**.
```
Comment (asphaltbuffet) [inline CODE_OF_CONDUCT.md:62]: Not sure what email address to use here... or maybe just say to create an issue in repo? No idea. Probably overthinking it for this project...

File: README.md [hunk 4]
```diff
@@ -1 +1,75 @@
-Passphrase to generate a random passphrase
+# Passphrase
+
+[![Keep a Changelog](https://img.shields.io/badge/changelog-Keep%20a%20Changelog-%23E05735)](CHANGELOG.md)
+[![GitHub Release](https://img.shields.io/github/v/release/asphaltbuffet/passphrase)](https://github.com/asphaltbuffet/passphrase/releases)
```
Comment (asphaltbuffet) [inline README.md]:
```suggestion
[![GitHub Release](https://img.shields.io/github/v/release/milligan22963/passphrase)](https://github.com/milligan22963/passphrase/releases)
```

File: README.md [hunk 5]
```diff
@@ -1 +1,75 @@
...
+[![Go Reference](https://pkg.go.dev/badge/github.com/asphaltbuffet/passphrase.svg)](https://pkg.go.dev/github.com/asphaltbuffet/passphrase)
```
Comment (asphaltbuffet) [inline README.md]:
```suggestion
[![Go Reference](https://pkg.go.dev/badge/github.com/milligan22963/passphrase.svg)](https://pkg.go.dev/github.com/milligan22963/passphrase)
```

File: README.md [hunk 6]
```diff
@@ -1 +1,75 @@
...
+[![go.mod](https://img.shields.io/github/go-mod/go-version/asphaltbuffet/passphrase)](go.mod)
```
Comment (asphaltbuffet) [inline README.md]:
```suggestion
[![go.mod](https://img.shields.io/github/go-mod/go-version/milligan22963/passphrase)](go.mod)
```

File: README.md [hunk 7]
```diff
@@ -1 +1,75 @@
...
+[![LICENSE](https://img.shields.io/github/license/asphaltbuffet/passphrase)](LICENSE)
```
Comment (asphaltbuffet) [inline README.md]:
```suggestion
[![LICENSE](https://img.shields.io/github/license/milligan22963/passphrase)](LICENSE)
```

File: README.md [hunk 8]
```diff
@@ -1 +1,75 @@
...
+[![Build Status](https://img.shields.io/github/workflow/status/asphaltbuffet/passphrase/build)](https://github.com/asphaltbuffet/passphrase/actions?query=workflow%3Abuild+branch%3Amain)
```
Comment (asphaltbuffet) [inline README.md]:
```suggestion
[![Build Status](https://img.shields.io/github/workflow/status/milligan22963/passphrase/build)](https://github.com/milligan22963/passphrase/actions?query=workflow%3Abuild+branch%3Amain)
```

File: README.md [hunk 9]
```diff
@@ -1 +1,75 @@
...
+[![Go Report Card](https://goreportcard.com/badge/github.com/asphaltbuffet/passphrase)](https://goreportcard.com/report/github.com/asphaltbuffet/passphrase)
```
Comment (asphaltbuffet) [inline README.md]:
```suggestion
[![Go Report Card](https://goreportcard.com/badge/github.com/milligan22963/passphrase)](https://goreportcard.com/report/github.com/milligan22963/passphrase)
```

File: README.md [hunk 10]
```diff
@@ -1 +1,75 @@
...
+[![Codecov](https://codecov.io/gh/asphaltbuffet/passphrase/branch/main/graph/badge.svg)](https://codecov.io/gh/asphaltbuffet/passphrase)
```
Comment (asphaltbuffet) [inline README.md]:
```suggestion
[![Codecov](https://codecov.io/gh/milligan22963/passphrase/branch/main/graph/badge.svg)](https://codecov.io/gh/milligan22963/passphrase)
```

File: go.mod [hunk 11]
```diff
@@ -1,5 +1,10 @@
-module github.com/milligan22963/passphrase
+module github.com/asphaltbuffet/passphrase
```
Comment (asphaltbuffet) [inline go.mod]: this needs to stay as your repo
```suggestion
module github.com/milligan22963/passphrase
```

File: main.go [hunk 12]
```diff
@@ -1,9 +1,9 @@
 /*
-  Copyright © 2022 DW Milligan dwm@afmsoftware.com
+Copyright © 2022 DW Milligan dwm@afmsoftware.com
 */
 package main
 
-import "github.com/milligan22963/passphrase/cmd"
+import "github.com/asphaltbuffet/passphrase/cmd"
```
Comment (asphaltbuffet) [inline main.go]:
```suggestion
import "github.com/milligan22963/passphrase/cmd"
```

**Review by asphaltbuffet — COMMENTED** [review]
i think it's ready? found all the spots where need to be careful with repo between forks.

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (milligan22963) [reply CODE_OF_CONDUCT.md:62]: Perhaps can put it at code@thepagefault.com

**Pull request merged**